contracts/src/SubnetActorDiamond.sol

```diff
@@ -25,18 +25,18 @@ contract SubnetActorDiamond {
     using SupplySourceHelper for SupplySource;
 
     struct ConstructorParams {
-        SubnetID parentId;
-        address ipcGatewayAddr;
-        ConsensusType consensus;
         uint256 minActivationCollateral;
+        uint256 minCrossMsgFee;
         uint64 minValidators;
         uint64 bottomUpCheckPeriod;
-        uint8 majorityPercentage;
+        address ipcGatewayAddr;
         uint16 activeValidatorsLimit;
-        uint256 minCrossMsgFee;
+        uint8 majorityPercentage;
+        ConsensusType consensus;
```

Comment:
Why is this `ConsensusType` required?

Reply:
Do you mean why we need this field if we have only one consensus protocol?

Reply:
The only value it has is `Fendermint`; I was just wondering what it really affects, and whether it's right to have such an enum. I don't know how to extend it. 

Sorry, I was lazy to go look; if it's "complicated" that's okay, I thought I'd ask.